start.py

```diff
@@ -724,6 +760,99 @@ def _configure_kudu(deployment, cluster, kudu_version):
                                                                configs)
 
 
+def _setup_ssl(cluster, product_list, ssl_authentication_required):
+    _setup_ssl_ca_authority(cluster)
+    if not ssl_authentication_required:
```

Comment:
Nit: let's flip this logic. Use `if ssl_authentication_required` as your check and then have the else for the other case. Also, in theory, we could remove a line of duplicate code by putting the `for service in service_list` at the top level and then doing the `if` check inside of that.

Reply:
Done.